.pipelines/ci.yml

```diff
@@ -473,6 +473,70 @@ stages:
         python -m pytest --ignore=test_cv2.py --ignore=test_tools_add_pre_post_processing_to_model.py . --verbose
       displayName: Run python test
 
+- stage: WindowsCUDABuilds
+  dependsOn: []
+  jobs:
+  - job: WindowsC
+    pool:
+      name: 'onnxruntime-extensions-Win2022-GPU-A10'
+    variables:
+      ort.version: '1.16.1'
```

Comment:
I suggest making a rule: all variable names must only use upper case letters and underscore . Because all such variables will be converted to environment variables. If you have a pipeline variable `ort.version`, it will become an env var `ORT_VERSION` that can be accessed from any build step. However, the two names are different. The differences could cause confusions.  see https://learn.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#environment-variables for more details.